test/unit/creator.js

```diff
@@ -0,0 +1,130 @@
+module.exports =  {
```

Comment:
Question: For consistencies sake; Do we just wanna move to inline html strings? I think it is more readable and that benefit definitely outweighs performance in tests, imo.